src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -68,6 +68,14 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		this._model.updateActiveCell(undefined);
 	}
 
+	@HostListener('document:keydown.meta.a', ['$event'])
```

Comment:
Also - what if the user changes their keybinding for select all? We should either respect the setting the user has for editors or at least make our own.

Reply:
If they do, it only applies to editors and the changes I made is on an angular component that doesn't have an editor. So probably we can document that somewhere? (Although I think we can agree that copy/paste/selectAll are something that users wouldn't want to change)